pkg/lvmdriver/services/identity_service.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"context"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+	"k8s.io/klog/v2"
+)
+
+// IdentityService handles requests from the container orchestrator
+// to determine readiness, capabilities, and identify the driver
+type IdentityService struct {
+	csi.UnimplementedIdentityServer
+	name         string
+	version      string
+	capabilities []csi.PluginCapability_Service_Type
+	ready        func() (bool, error)
+}
+
+// NewIdentityService returns a new IdentityService.
+//
+// ready is a function to check the plugin status.
+// It should return non-nil error if the plugin is not healthy.
+// If the plugin is not yet ready, it should return (false, nil).
+// Otherwise, return (true, nil).
+func NewIdentityService(name string, version string, ready func() (bool, error)) csi.IdentityServer {
+	return &IdentityService{
+		ready:   ready,
+		name:    name,
+		version: version,
+		capabilities: []csi.PluginCapability_Service_Type{
```

Comment:
Why not have empty capabilities? Is this required right now?

Reply:
I think Unknown is fine for now, it gives us a hard value to test against and will show once it's removed and real capabilities are added and the tests start failing